src/util.rs

```diff
@@ -34,6 +34,16 @@ pub fn document() -> web_sys::Document {
         .expect("Can't find the window's document")
 }
 
+/// Convenience function to access the `web_sys::HtmlDocument`.
+pub fn html_document() -> web_sys::HtmlDocument {
+    to_html_document(document())
+}
+
+/// Convenience function to convert a `Document` to an `HtmlDocument`.
+pub fn to_html_document(doc: web_sys::Document) -> web_sys::HtmlDocument {
```

Comment:
- What's your use-case, please? (Can it be private?)
- Implementing `From<Document> for HtmlDocument` is probably not possible, but can we at least rename this function to something more expressive (like `document_to_html_document`)? Or more elegant solution could be to write trait like `ToHtmlDocument` with one function `to_html_document`, implement it for `Document` and add it into Seed's prelude.

Reply:
>* What's your use-case, please? 

I need to check the cookies of the document
> (Can it be private?)

yes, this one can be private

>* Implementing `From<Document> for HtmlDocument` is probably not possible, but can we at least rename this function to something more expressive (like `document_to_html_document`)?

yes, that's possible :)

> Or more elegant solution could be to write trait like `ToHtmlDocument` with one function `to_html_document`, implement it for `Document` and add it into Seed's prelude.

that's also nice.
What do you prefer?

Reply:
> I prefer `From<Document> for HtmlDocument` of course I'd also prefer this but that's not allowed in Rust. We have either to wrap one of both into a new struct or define or create feature request for `web_sys`

Reply:
> of course I'd also prefer this but that's not allowed in Rust.

I thought so but then you wrote "yes, that's possible :)"..  a little misunderstanding :)

I have no preference between renaming and adding trait - the first is simple and explicit, the second one doesn't pollute public API. Or choose another solution if you find it more suitable.